engine/Default/feature_request.php

```diff
@@ -3,43 +3,44 @@
 	create_error('Feature requests are currently not being accepted.');
 }
 
-$template->assign('PageTopic','Feature Request');
-
-if(!isset($var['Status'])) {
-	SmrSession::updateVar('Status', 'Opened');
+if (!isset($var['category'])) {
+	SmrSession::updateVar('category', 'New');
 }
+$thisStatus = statusFromCategory($var['category']);
 
-$container = $var;
-$container['Status'] = 'Implemented';
-$template->assign('ViewImplementedFeaturesHref',SmrSession::getNewHREF($container));
-
-$container = $var;
-$container['Status'] = 'Opened';
-$container['ShowOld'] = true;
-$template->assign('ShowOldFeaturesHref',SmrSession::getNewHREF($container));
+$template->assign('PageTopic', 'Feature Requests - ' . $var['category']);
 
-$container = $var;
-$container['Status'] = 'Rejected';
-$template->assign('ShowRejectedFeaturesHref',SmrSession::getNewHREF($container));
+// Feature Requests show up as new for this many days
+$newRequestDays = 30;
```

Comment:
This can be defined as a constant as there's nowhere that changes it

Reply:
Done.